topics/persistence.md

```diff
@@ -6,7 +6,13 @@ Redis Persistence
 Redis provides a different range of persistence options:
 
 * **RDB** (Redis Database): The RDB persistence performs point-in-time snapshots of your dataset at specified intervals.
-* **AOF** (Append Only File):  The AOF persistence logs every write operation received by the server, that will be played again at server startup, reconstructing the original dataset. Commands are logged using the same format as the Redis protocol itself, in an append-only fashion. Redis is able to rewrite the log in the background when it gets too big.
+* **AOF** (Append Only File): The AOF persistence logs every write operation received by the server, that will be played again at server startup, reconstructing the original dataset. Commands are logged using the same format as the Redis protocol itself, in an append-only fashion. Redis is able to rewrite the log in the background when it gets too big.    
+
+  Since Redis 7.0.0, Redis supports the Multi Part AOF mechanism.
+  That is, the original single AOF file is split into base file (at most one) and incremental files (there may be more than one).
+  The base file represents an initial (RDB or AOF format) snapshot of the data present when the AOF is rewritten.
+  The incremental files contains incremental commands since the last base AOF file was created. All these files are put in a separate folder and are tracked by a manifest file.
```

Comment:
```suggestion
  <br/>Since Redis 7.0.0, Redis supports a multi part AOF mechanism.
  That is, the original single AOF file is split into base file (at most one) and incremental files (there may be more than one).
  The base file represents an initial (RDB or AOF format) snapshot of the data present when the AOF is rewritten.
  The incremental files contains incremental commands since the last base AOF file was created. All these files are put in a separate directory and are tracked by a manifest file.
```

Reply:
OK